src/python/setup.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+
+def readme():
+    with open("README.md") as desc:
+        return desc.read()
+
+
+setup(
+    name="hirte",
+    version="1.0.0",
```

Comment:
I'd probably start  with with `0.1.0` version, `1.0.0` seems too *stable* for initial release ...